### PR TITLE
Add civi

### PR DESCRIPTION
**Pull request by dborstelmann**
@DCalliet DON'T MERGE THIS.

Frontend is basically there.  Needs to be tested.  Will test tomorrow, please check how this is supposed to interact with backend so I know if I am doing this right.

### CODE REVIEW

File: civiwiki/frontend/static/views/add_civi_view.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            data: {
+                title: title,
+                category: category,
+                topic: topic,
+                type: ics,
+                body: body,
+                creator: '5'
+            },
+            success: function () {
+
```
Comment (DCalliet) [inline civiwiki/frontend/static/views/add_civi_view.js:46]: maybe redirect to db_view, focused on the current civi you just created?

Comment (dborstelmann) [inline civiwiki/frontend/static/views/add_civi_view.js:46]: I'll probably do this after the view civi thing gets built, right now I wanted people to be able to make a lot of civis quickly which is why I left it on the same page.

**Comment by DCalliet** [comment]
@dborstelmann this is getting stale too homie, this should be in master asap.

**Comment by dborstelmann** [comment]
@DCalliet There were some flix on this, that's what I was solving.  Done now.

**Pull request merged**